packages/daimo-api/src/api/getAccountHistory.ts

```diff
@@ -253,6 +267,14 @@ async function getNamedAccountsFromClogs(
     await Promise.all([...addrs].map((addr) => nameReg.getEAccount(addr)))
   ).filter((acc) => hasAccountName(acc));
 
+  // Map Landline liquidation addresses to the corresponding bank account
+  for (const landlineAccount of landlineAccounts) {
+    namedAccounts.push({
+      addr: landlineAccount.liquidationAddress,
+      name: getLandlineAccountName(landlineAccount),
+    });
```

Comment:
is this needed given the special handling for landline accounts?

Reply:
this is for backcompat. enables us to show "Chase ****1234" for withdrawals in old apps which don't have the `landlineAccountCache` in the mobile code